### PR TITLE
Typo and grammar fixes to documenting.rst "Translating" section

### PR DESCRIPTION
**Pull request by bgrant**
This consists only of minor typo fixes and grammar suggestions for the `Translating` section of `documenting.rst`.  Please let me know if you would like an issue made.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@bgrant

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by hugovk** [comment]
@bgrant Looks good, would you like to resolve the conflicts?

**Comment by bgrant** [comment]
Done!  Thanks for your review!

**Review by hugovk — APPROVED** [review]

**Comment by bgrant** [comment]
Fixed more conflicts.  Anything else I need to do to get this merged?  Thanks.

File: documenting.rst [hunk 1]
```diff
@@ -1749,62 +1749,62 @@ and there's a `Libera.Chat IRC <https://libera.chat/>`_ channel,
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Consensus is to work on current stable, you can then propagate your
-translation from a branch to another using `pomerge
+Consensus is to work on current stable. You can then propagate your
+translation from one branch to another using `pomerge
 <https://pypi.org/project/pomerge/>`__.
 
 
+
```
Comment (JelleZijlstra) [inline documenting.rst:1760]: two blank lines are enough
```suggestion
```

**Review by JelleZijlstra — COMMENTED** [review]
Thanks! I can merge this.

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**